R/plot.R

```diff
@@ -102,10 +104,10 @@ plot_surv <- function(time_fixef, time_rng, x_range, fixef_vary, vcov_vary,
 
     exp(qnorm(p = ps, mean = log_haz, sd = sqrt(log_haz_var)))
   }))
-  matplot(tis, hazs, lty = 1, type = "l", col = "black", bty = "l",
-          xlab = xlab, xaxs = "i", yaxs = "i", ylab = ylab,
+  matplot(tis, hazs, type = "n", xlab = xlab, bty = "l", ylab = ylab,
           ylim = range(hazs, 0), ...)
   grid()
+  matplot(tis, hazs, lty = 1, type = "l", col = "black", add = TRUE, ...)
```

Comment:
Same as for the comment for the `points(xs, mea, type = "l")` call.

Reply:
The reason here is the same as above, to have the grid _behind_ the actual plotted curves.